_episodes/71.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: episode
+title: "71: Fake it real good"
+date: 2025-02-27
+tags: trino faker data generation testing learning SQL
+youtube_id: ""
+wistia_id: ""
+sections:
+- time: 0:00
+  title: Introduction
+- time: 1:08
+  title: Trino 471 release update
+introduction: |
+  Manfred Moser and Cole Bowden are joined by Jan Waś to learn about the new 
+  Faker connector and how you can use it to emulate data that does not exist on
+  any storage, how you can shape it as you need, and how you can then learn real
+  SQL, build real reports, and make some real charts -  all with fake data.
+---
+
+## Host
+
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Director/Open
+  Source Engineering at [Starburst]({{site.url}}/users.html#starburst) -
+  [@simpligility](https://x.com/simpligility)
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Firebolt](https://www.firebolt.io/) 
+  
+## Guests
+
+* [Jan Waś](https://www.linkedin.com/in/janwas/), 
+Software Engineer at [Starburst]({{site.url}}/users.html#starburst)
+
+## Releases
+
+Following are some highlights of the Trino releases since episode 69:
+
+[Trino 471]({{site.baseurl}}/docs/current/release/release-471.html)
+
+* Add [AI functions]({{site.url}}/docs/current/functions/ai.html) for textual
+  tasks on data using OpenAI, Anthropic, or other LLMs using Ollama as backend.
+* Add support for JSON logging format to console (useful in containers..).
```

Comment:
```suggestion
* Add support for formatting JSON logging to console (useful in containers).
```

Reply:
thats sounds wrong .. the whole log is JSON now if desired..

Reply:
Maybe

Add support for logging output to the console in JSON format ??